server/apps/memory-app/README.md

```diff
@@ -32,6 +32,15 @@ keytool -genkey -alias james -keyalg RSA -keystore keystore
 docker run -v $PWD/keystore:/root/conf/keystore apache/james:memory-latest
 ```
 
+In the case of quick start James without manually creating a keystore (e.g. for development),
+James memory server will auto-generate keystore file with the default setting that is declared in `jmap.properties` (tls.keystoreURL, tls.secret)
+Hence just run docker command:
+
+```
+docker run apache/james:memory-latest
```

Comment:
Why not `--generate-keystore` ?

I advocate to do exactly the same that other servers...

Reply:
Base on your suggestion: "We can make start-dev the default for memory"

Reply:
Well, now that I see this, I really wonder why we do anything special for memory ;-)